docs/volunteering/writing/editing-bobadocs.md

```diff
@@ -0,0 +1,158 @@
+- `static`: containing static site data, such as images
+
+We want to edit a page in the site, so click `docs`.
+
+![a screenshot of the git repo, with a red arrow pointing to the docs folder](/img/volunteer/bobadocsnav1.jpg)
+
+You'll notice four new folders: `engineering`, `product`, `users`, and `volunteer`. Each of these folders corresponds to one of the guides on BobaDocs. Click the folder for the guide with the page you want to edit.
+
+Once there, you'll notice something a little odd:
+
+![a screenshot of the git repo compared to the sidebar on BobaDocs. lines are drawn between items with similar names](/img/volunteer/bobadocsnav2.jpg)
+
+Some of the names of these things are QUITE SIMILAR. This isn't a coincidence! Docusaurus generates the sidebar for BobaDocs based on the files and folders present: each file is a page on the site, while each folder is a section on the sidebar.
+
+Armed with this information (and a little investigation), I determine that the page I want to edit, Codebase Overview, is named `intro.md` in the repo. I click it.
+
+![a screenshot of intro.md previewed on github. the edit button is highlighted](/img/volunteer/bobadocsnav3.jpg)
+
+Doing so has led me to an overview of Codebase Overview. Near the top right of the file preview is a button shaped like a pencil. Hovering over it will give you its details: "Edit this file". Click it, and you'll be brought to GitHub's built-in editor.
+
+### Method 2: Go directly from the page you want to edit
+
+At the bottom of every page of documentation is a green link labeled "Edit this page". Click it, and you'll get a 404 error, courtesy of those editing permissions mentioned earlier. Luckily, there's more here than you may think. Look at the URL:
+
+```
+https://github.com/essential-randomness/bobaboard/edit/main/website/docs/engineering/intro.md
+```
+
+I got this URL by clicking the link at the bottom of the [Engineering Guide's Codebase Overview page](../../engineering/intro). This URL follows a logical format, which can be described as such:
+
+```
+https://github.com/[USERNAME]/[REPO NAME]/edit/[BRANCH NAME]/[PATH TO FILE]
+```
+
+That means that in order to get to the editing page, all we have to do is replace the values in the brackets with the values corresponding to our copy of the file. Here are the steps you need to do that:
+
+1. Change `[USERNAME]` (default `essential-randomness`) to your GitHub username
+
+2. Change `[REPO NAME]` (default `bobaboard`) to `bobadocs`
+
+3. Change `[BRANCH NAME]` (default `main`) to the name of the branch you made earlier, with dashes replacing any spaces
+
+4. Delete `website/` from `[PATH TO FILE]`
+
+If I use my own information to make these changes, I get the URL:
+
+```
+https://github.com/rbot-github/bobadocs/edit/new-branch/docs/engineering/intro.md
+```
+
+Loading that URL, I now have GitHub's editor open, and can make my changes!
+
+## Step 2: Make your edits
```

Comment:
I stopped here and haven't reviewed the rest. I'll pick it back up once you do a first pass :)